Dockerfile.alpine

```diff
@@ -32,6 +32,14 @@ RUN cd /tmp && \
 
 RUN update-ca-certificates
 
+# Librdkafka for event service dependents
+RUN curl -L https://github.com/edenhill/librdkafka/archive/v1.2.1.zip > librdkafka.zip && \
+  unzip librdkafka.zip && \
+  cd librdkafka-1.2.1 && \
+  ./configure --prefix /usr && \
+  make && \
+  make install
```

Comment:
`make install` with no prefix doesn't require superuser?

Reply:
apparently not. `sudo` was originally required as one of the additions to `apk install` but turns out it's unnecessary. Though I do wonder if that'll somehow affect the linking of the python package to `librdkafka.h`

Reply:
Nevermind, works fine 🤷‍♂

Reply:
this runs as root